presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraClientConfig.java

```diff
@@ -379,4 +394,16 @@ public CassandraClientConfig setSpeculativeExecutionDelay(Duration speculativeEx
         this.speculativeExecutionDelay = speculativeExecutionDelay;
         return this;
     }
+
+    public ProtocolVersion getProtocolVersion()
+    {
+        return protocolVersion;
+    }
+
+    @Config("cassandra.protocol-version")
+    public CassandraClientConfig setProtocolVersion(String version)
```

Comment:
Just define the parameter as `ProtocolVersion`. The configuration framework does the conversion by itself.